javascript/javascript1/week3/Homework/Exercise5.js

```diff
@@ -14,35 +15,57 @@ function addActivity(date, activity, duration) {
   activities.push(newActivity);
```

Comment:
Great you have also passed a default date value. It's not something we have introduced to class yet but you could actually shorten this down using [shorthand syntax for object properties and values](https://www.geeksforgeeks.org/shorthand-syntax-for-object-property-value-in-es6/).

This could be written as

`activities.push({ date, activity, duration }); // <-- this both passes the key date and gives it the value of date & same with activity & duration`